packages/@react-spectrum/dialog/test/DialogTrigger.test.js

```diff
@@ -35,7 +36,16 @@ describe('DialogTrigger', function () {
   });
 
   afterEach(() => {
-    jest.runAllTimers();
+    // Ensure we close any dialogs before unmounting to avoid warning.
+    let dialog = document.querySelector('[role="dialog"]');
```

Comment:
could probably use `screen.getByRole('dialog')` instead

Reply:
what is screen? Any benefits or about the same?